modules/models/src/main/java/org/eclipse/xpanse/modules/models/monitor/Metric.java

```diff
@@ -30,6 +31,10 @@ public class Metric {
     @Schema(description = "The type of the metric.")
     MetricType type;
 
+    @NotNull
+    @Schema(description = "The resources type of the monitor.")
```

Comment:
@jinyangyang222 description should be "the resource type  of the metric."

Reply:
ok, fixed.